CHANGELOG.next.asciidoc

```diff
@@ -152,6 +152,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Fix spooling to disk blocking infinitely if the lock file can not be acquired. {pull}15338[15338]
 - Fix `metricbeat test output` with an ipv6 ES host in the output.hosts. {pull}15368[15368]
 - Fix `convert` processor conversion of string to integer with leading zeros. {issue}15513[15513] {pull}15557[15557]
+- Fix panic while trying to send events but the connection is closed {pull}15568[15568]
```

Comment:
Let's add some info where the panic occurs.

'Fix panic in the Logstash output when trying to send events to closed connection'

Reply:
Entry renamed.